protocols/bftcosi/bftcosi.go

```diff
@@ -15,13 +14,16 @@ import (
 // VerificationFunction can be passes to each protocol node. It will be called
 // (in a go routine) during the (start/handle) challenge prepare phase of the
 // protocol. The passed message is the same as sent in the challenge phase.
-type VerificationFunction func(Msg []byte) bool
+type VerificationFunction func(Msg []byte, Data []byte) bool
```

Comment:
Damn I don't think so. that would be a really good thing to do, even better if we can remove the data part here.